### PR TITLE
Small updates to the release notes

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

File: doc/RELEASE_NOTES.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ Alternatively, one can specify `fedoraproject_username` on the command line, rat
...
+While we don't include installation instructions for the new `node-exporter` and `dcgm` tools in the published documentation, you can find a manual installation procedure for the Prometheus "node_exporter" and references to the Nvidia "DCGM" documentation in the [`agent/tool-scripts/README`](https://github.com/distributed-system-analysis/pbench/blob/v0.71.0/agent/tool-scripts/README.md).
 
-Container images built using the above RPMs are available in the [Pbench](https://quay.io/organization/pbench) organization in the Quay.io container image repository using tags `beta`, `v0.71.0-2`, and `9d90a97cc`.
+Container images built using the above RPMs are available in the [Pbench](https://quay.io/organization/pbench) organization in the Quay.io container image repository using tags `latest`, `v0.71.0-4`, and `XXXXXXXXX`.
```
Comment (webbnh) [inline doc/RELEASE_NOTES.md:61]: Do we need/expect to replace the `X`'s before merging this change?

**Review by webbnh — DISMISSED** [review]
Were we going to add a headline about having to reregister the tools?  (Right there with the default tool set deprecation notice?)  The existing requirement is still buried pretty deep.

**Comment by portante** [comment]
> Were we going to add a headline about having to reregister the tools? (Right there with the default tool set deprecation notice?) The existing requirement is still buried pretty deep.

Yes, thank you for reminding me!

File: doc/RELEASE_NOTES.md (hunk 1 above)
Comment (portante) [reply doc/RELEASE_NOTES.md:61]: Yes, I was waiting for the official tag to be made before I did that.  I'll update that now.

**Review by portante — COMMENTED** [review]
Okay, looks ready to go!  I have already deployed this version to make sure it formats okay.

File: doc/RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
 This is a very *significant* "minor" release of the pbench-agent code base, primarily to deliver the new "Tool Meister" sub-system.
 
-_*NOTE WELL*_: The notion of a "default" tool set is being deprecated and will be removed in the upcoming Pbench Agent v1.0 release, along with the addition of a few named tool sets.  See below.
+_*NOTE WELL*_:
+
+ * The notion of a "default" tool set is being deprecated and will be removed in the upcoming Pbench Agent v1.0 release, along with the addition of a few named tool sets.  See "Default Tool Set is _*Deprecated*_; Named tool sets introduced" below.
```
Comment (dbutenhof) [inline doc/RELEASE_NOTES.md:5]: This sentence somewhat implies *removal* of "a few named tool sets". Maybe break up the sentence? "The notion of a 'default' tool set [...] will be removed ... To replace it, Pbench is introducing a set of named tool sets." ???

**Review by dbutenhof — APPROVED** [review]

File: doc/RELEASE_NOTES.md (hunk 2 above)
Comment (portante) [reply doc/RELEASE_NOTES.md:5]: Good point.

File: doc/RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,11 +1,13 @@
 This is a very *significant* "minor" release of the pbench-agent code base, primarily to deliver the new "Tool Meister" sub-system.
 
-_*NOTE WELL*_: The notion of a "default" tool set is being deprecated and will be removed in the upcoming Pbench Agent v1.0 release, along with the addition of a few named tool sets.  See below.
+_*NOTE WELL*_:
+
+ * The notion of a "default" tool set is being deprecated and will be removed in the upcoming Pbench Agent v1.0 release.  To replace it, the Pbench Agent is introducing the addition of a few named tool sets.  See "Default Tool Set is _*Deprecated*_; Named tool sets introduced" below.
```
Comment (dbutenhof) [inline doc/RELEASE_NOTES.md:5]: "introducing the addition of"? Maybe drop "the addition of"?

**Review by dbutenhof — APPROVED** [review]

**Review by webbnh — APPROVED** [review]
👍

**Pull request merged**